packages/code-du-travail-frontend/pages/contribution/[slug].tsx

```diff
@@ -16,36 +19,45 @@ interface Props {
   breadcrumbs: Breadcrumb[];
   description: string;
   title: string;
+  slug: string;
   content;
   answers;
   relatedItems: Array<any>;
 }
 
-function PageContribution(props: Props): JSX.Element {
-  const buildTitleAndDescription = (
-    breadcrumbs,
-    conventionAnswer,
-    title,
-    description
-  ) => {
-    if (breadcrumbs && breadcrumbs.length > 0 && conventionAnswer) {
-      const titleWithThemeAndCC =
-        breadcrumbs[breadcrumbs.length - 1].label +
-        " - " +
-        conventionAnswer.shortName;
-      return {
-        description: title + " " + description,
-        title: titleWithThemeAndCC,
-      };
-    }
+const buildTitleAndDescription = (
+  breadcrumbs,
+  conventionAnswer,
+  title,
+  description
+) => {
+  if (breadcrumbs && breadcrumbs.length > 0 && conventionAnswer) {
+    const titleWithThemeAndCC =
+      breadcrumbs[breadcrumbs.length - 1].label +
+      " - " +
+      conventionAnswer.shortName;
     return {
-      description,
-      title,
+      description: title + " " + description,
+      title: titleWithThemeAndCC,
     };
+  }
+  return {
+    description,
+    title,
   };
+};
+const SLUG_FOR_POC_GENERIC = ["les-conges-pour-evenements-familiaux"];
 
-  const { breadcrumbs, title, answers, description, relatedItems, content } =
-    props;
+function PageContribution(props: Props): JSX.Element {
```

Comment:
JSX.Element est deprecated (https://stackoverflow.com/questions/76224848/what-is-the-correct-return-type-replacement-for-jsx-element-after-the-global-jsx)

```suggestion
function PageContribution(props: Props): React.ReactElement {
```